app/Services/MQTTNotifier.php

```diff
@@ -0,0 +1,36 @@
+<?php declare(strict_types = 1);
+
+namespace App\Services;
+
+// Helpers
+use App\Helpers\MQTT;
+
+// Models
+use App\Models\Config;
+
+class MQTTNotifier implements Notifier
+{
+    private $client;
+    private $topic;
+
+    public function __construct(MQTT $client, string $topic)
+    {
+        $this->client = $client;
+        $this->topic = $topic;
+    }
+
+    public function new(Config $config)
+    {
+        $this->client->publish($this->topic, "new:{$config->slack_token}");
+    }
+
+    public function update(string $token, Config $config)
```

Comment:
Since token is present in $config, isn't it unnecessary to require both as arguments to this function?